definitions/features/redis.rb

```diff
@@ -0,0 +1,34 @@
+class Features::Redis < ForemanMaintain::Feature
+  SCL_NAME = 'rh-redis5'.freeze
+
+  metadata do
+    label :redis
+
+    confine do
+      # Luckily, the service name is the same as the package providing it
+      server? && find_package(service_name)
```

Comment:
Given block passed as argument to `metadata` and its execution happens outside of this feature scope so it will raise an error for `service_name`.

Either you can declare one more constant or use "#{SCL_NAME}-redis" directly.

Reply:
Oh, forgot about that. Fixed in a slightly different way